examples/layers/JSLayers/IGN_MNT.js

```diff
@@ -0,0 +1,73 @@
+itowns.viewer.addElevationLayer({
```

Comment:
that's right, we should probably remove all the js layer files as we don't use it. I will do it !